dark/assets/css/styles.css

```diff
@@ -1051,6 +1051,12 @@ html {
   z-index: -1;
 }
 
+#contact .contact-text {

```

Comment:
Can you add these css properties to the below `.contact-text` class @Rishav1707

Reply:
Done

Reply:
Now remove the code from here @Rishav1707

Reply:
Do the above mentioned changes @Rishav1707 
```suggestion
```